packages/loader/container-loader/src/container.ts

```diff
@@ -998,10 +997,6 @@ export class Container
 			0x0d2 /* "resolved url should be valid Fluid url" */,
 		);
 		assert(!!this._protocolHandler, 0x2e3 /* "Must have a valid protocol handler instance" */);
-		assert(
-			this._protocolHandler.attributes.term !== undefined,
```

Comment:
Normally this assert could cause us problems after the server drops `term` from the attributes it provides (i.e. new server + old loader).  However, the `ProtocolOpHandler` stuffs a term of "1" on if not provided so that should doctor it up (new server + old loader + old protocol handler is fine).

Customers using a custom `protocolHandlerBuilder` and expecting it to be used with an old loader must continue doctoring the `term` (as they must already do today, either via using the older `ProtocolOpHandler` or in their own implementation).